src/Illuminate/Foundation/Application.php

```diff
@@ -221,9 +222,8 @@ public function __construct($basePath = null)
      * Begin configuring a new Laravel application instance.
      *
      * @param  string|null  $baseDirectory
-     * @return \Illuminate\Foundation\ApplicationBuilder
```

Comment:
This class does not exist, `Configuration` was missing

Reply:
![type-ezgif com-cut](https://github.com/laravel/framework/assets/924196/257597fd-8aea-4a98-9497-d87420d0ed70)

Reply:
Add return type into doc